connectors/citrus-openapi/src/main/java/org/citrusframework/openapi/actions/OpenApiClientActionBuilder.java

```diff
@@ -136,11 +136,11 @@ public TestActionBuilder<?> getDelegate() {
      */
     @Override
     public void setReferenceResolver(ReferenceResolver referenceResolver) {
```

Comment:
```suggestion
    public void setReferenceResolver(@Nullable ReferenceResolver referenceResolver) {
```

would this be possible (is the class accessible)?